dm/docs/RFCS/20220124_enhance_task_manageability.md

```diff
@@ -0,0 +1,127 @@
+| list    | `dmctl ddl-lock list --task-name="test"`                     | --task-name              | show shard-ddl locks information for a task. |
+| unlock  | `dmctl ddl-lock unlock --task-name="test" --lock-id="lock1"` | --task-name, --lock-name | force unlock un-resolved DDL locks.          |
+
+### dmctl commands for member
+
+| command                    | Full Syntax Example                                                 | Flags          | Description                                       |
+| -------------------------- | ------------------------------------------------------------------- | -------------- | ------------------------------------------------- |
+| member list                | `dmctl member list --name="master-1" --role="master/worker/leader"` | --name, --role | show members of current cluster by name and role. |
+| member offline             | `dmctl member offline --name="master-1"`                            | --name         | offline members of current cluster by name.       |
+| member evict-leader        | `dmctl member evict-leader --name="master-1"`                       | --name         | evict leader for master node.                     |
+| member cancel-evict-leader | `dmctl member cancel-evict-leader --name="master-1"`                | --name         | cancel evict leader for master node.              |
+
+### Optimised dmctl for interaction mode (optional)
+
+The current interaction of dmctl still has room for optimization, and I hope to take this opportunity to do some optimization of the interaction experience, focusing on the user can quickly select the command they want to enter through the keyboard, instead of entering commands by their memory.
+
+Here is a simple prototype demo:
+
+[![asciicast](https://asciinema.org/a/AVnOlghSu5d0YqjdOw9iruFdW.svg)](https://asciinema.org/a/AVnOlghSu5d0YqjdOw9iruFdW)
+
+## Breaking Changes for OpenAPI
+
+- `POST /api/v1/tasks` will be changed from creating and starting tasks to only creating tasks.
+- `DELETE /api/v1/tasks/{task-name}` will be changed from stopping task to stopping task and delete the meta data.
+- `POST /api/v1/tasks/{task-name}/pause` will be updated to `POST /api/v1/tasks/{task-name}/stop`
+- `POST /api/v1/tasks/{task-name}/resume` will be updated to `POST /api/v1/tasks/{task-name}/start`
+- `POST /api/v1/sources/{source-name}/pause-relay` will be deleted
+- `POST /api/v1/sources/{source-name}/resume-relay` will be deleted
+
+## Milestones
+
+### Milestone 1 - Implementation of the modified task state machine according to the design documentation
+
+This phase is mainly about implementing the DM-Master/DM-Worker internal logic.
+
+For tasks, the DM-Master's internal scheduling module needs to support the creation of a subtask in a stopped state, in addition to adapting additional parameters like `-start-time`,`--time-out` and so on.
```

Comment:
> For tasks, the DM-Master's internal scheduling module needs to support the creation of a subtask in a stopped state

what happens to the stopped subtask? user may easily override existing task(stopped)